packages/cascara/src/ui/Table/Table.test.js

```diff
@@ -81,6 +81,7 @@ describe('Table', () => {
           module: 'text',
         },
         {
+          'aria-label': 'Email',
           attribute: 'email',
           'data-testid': 'email',
```

Comment:
IMO we should actually not even be passing `'data-testid': 'email',` in these objects. Instead, we should just be using the selector to check the label.

Reply:
it makes sense, let me take them out